buildchain/buildchain/salt_tree.py

```diff
@@ -277,7 +277,7 @@ def _run(self) -> None:
     Path('salt/metalk8s/repo/configured.sls'),
     Path('salt/metalk8s/repo/deployed.sls'),
     Path('salt/metalk8s/repo/files/nginx.conf.j2'),
-    Path('salt/metalk8s/repo/files/90-metalk8s-registry-config.inc.j2'),
```

Comment:
Why this change? How is it related so `s/SHORT_VERSION/VERSION/`?

Reply:
Just removed the prefix as it's not needed to have those prefix in our salt tree

Reply:
No related to the VERSIOn switch then, so should be in a dedicated commit (where you could explain it's uneeded in the commit message).

Reply:
Done